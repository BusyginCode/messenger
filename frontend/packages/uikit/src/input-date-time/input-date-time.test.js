import React from 'react'
import { shallow } from 'enzyme'
import InputDateTime from './input-date-time'

describe('uikit/InputDateTime', () => {
  it('should render correctly', () => {
    expect(shallow(<InputDateTime />)).toMatchSnapshot()
  })
})
