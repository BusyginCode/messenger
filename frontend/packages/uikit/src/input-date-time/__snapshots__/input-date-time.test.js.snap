// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/InputDateTime should render correctly 1`] = `
<Box
  component="div"
>
  <Component
    className="datepicker"
    closeOnSelect={true}
    closeOnTab={true}
    dateFormat={true}
    defaultValue=""
    input={true}
    inputProps={Object {}}
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    onViewModeChange={[Function]}
    renderInput={[Function]}
    strictParsing={true}
    timeConstraints={Object {}}
    timeFormat={false}
    utc={false}
  />
  <Svg(calendar)
    className="icon"
    onClick={[Function]}
  />
</Box>
`;
